### DISCUSSION TITLE
HACS Support?

### DISCUSSION THREAD

**Discussion by corbanmailloux** [DurgNomis-drol/google_home_timers_card#14] [Ideas]
I'm curious if you'd be interested in adding support for this repository in HACS (https://hacs.xyz/), much like the [underlying custom component](https://github.com/leikoilja/ha-google-home) already does.

Thanks for what you're doing!

**Comment by DurgNomis-drol** [comment]
Yes that would properly be a good idea 😁

Our original plan was to deliver both of them through the same repository, but I don't think that is possible unfortunately.

Will add this to the todo list 😊

Thanks for the positive feedback

**Comment by DurgNomis-drol** [comment]
I have now added it to HACS, but only as a custom repository for now. I will try to get it into default soon, but some features is missing and I need to rewrite the script first. 😄

**Comment by corbanmailloux** [comment]
Wow, thanks for the quick turn-around!
